scripts/refresh_postgres.pl

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/perl -w
+
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# This script expects nginx access logs on STDIN
+# filtered by the app:
+# grep "Official Android App" nginx.access2.log | grep Scan > android_app.log
+
+use Modern::Perl '2017';
+
+use ProductOpener::Config qw/:all/;
+
+use LWP::UserAgent;
+
+$query_url =~ s/^\s+|\s+$//g;
+my $from = $ARGV[0] || '';
+my $path = "$query_url/importfrommongo?from=$from";
+print STDERR "Importing to $path\n";
+my $ua = LWP::UserAgent->new();
+$ua->get($path);
```

Comment:
Is this query returning quickly ? (maybe it should)

Reply:
This does wait until the refresh has been completed before returning at the moment. It will be quick if it is an incremental update, but takes about 1.5 hours to do a full load.